test/storage/large_garbage_collector_test.cpp

```diff
@@ -304,4 +304,76 @@ TEST_F(LargeGCTests, TPCCishHighThreadWithGC) {
     EndGC();
   }
 }
+
+// This test attempts to simulate a long running read-only OLAP query, with OLTP
+// queries in a TPC-C-like scenario.
+// NOLINTNEXTLINE
+TEST_F(LargeGCTests, OLAPAndTPCCishWithGC) {
+  const uint32_t txn_length = 5;
+  const std::vector<double> update_select_ratio = {0.4, 0.6};
```

Comment:
It is a common select ratio. Since the update to select ratio is low, the expectation is that the number of aborts will be low, but there will still be a considerable number of updates happening thus increasing the version chains' lengths.